rancid-tomatillos/src/Card/Card.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
First test! Seems we learned from the lesson today how we can start writing integration tests for App now too